cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -35,12 +29,8 @@ def test_embedded_ansible_enable(enabled_embedded_appliance):
     assert wait_for(func=lambda: enabled_embedded_appliance.is_embedded_ansible_running, num_sec=30)
     assert wait_for(func=lambda: enabled_embedded_appliance.is_rabbitmq_running, num_sec=30)
     assert wait_for(func=lambda: enabled_embedded_appliance.is_nginx_running, num_sec=30)
-    if not enabled_embedded_appliance.is_pod:
-        endpoint = 'ansibleapi'
-        imprint = 'Ansible Tower'
-    else:
-        endpoint = 'api'
-        imprint = 'AWX'
+    endpoint = "api" if enabled_embedded_appliance.is_pod else "ansibleapi"
+    imprint = 'AWX'
```

Comment:
@sbulage 
I think ```imprint``` is used once here. 
So we don't need this extra variable; we can directly write ```p='AWX'```.

Reply:
Removed it.